src/app/operators/dialogs/draw-features/ol-draw-features.component.ts

```diff
@@ -67,24 +85,22 @@ export class OlDrawFeaturesComponent implements OnInit, OnDestroy {
         if (olSource.getFeatures().length > 0) {
             this.createAndAddOperatorFromSource(olSource);
         } else {
-            this.notificationService.info('Empty layer skipped.')
+            this.notificationService.info('Empty layer skipped.');
         }
     }
 
     private createAndAddOperatorFromSource(olSource: OlVectorSource) {
-
-        let resultType: ResultType;
         let resultSymbology: AbstractVectorSymbology;
+        let clustered = false;
 
         switch (this.selectedFeatureType) {
-            case 'Point':
-                resultType = ResultTypes.POINTS;
+            case ResultTypes.POINTS:
                 resultSymbology = ComplexPointSymbology.createClusterSymbology({
                     fillRGBA: this.randomColorService.getRandomColorRgba(),
                 });
+                clustered = true;
```

Comment:
I don't think that the default style for points from user input should be clustered

Reply:
I thought it was useless to use a cluster symbology without having this parameter set to be true. But I could change it to unclustered, but then with SimplePointSymbology.

Reply:
:+1: